util/test/prediff-for-incremental-warning

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+#
+# This script is a system-wide prediff for use when incremental compilation is enabled.
+# If --incremental is used along with --fast it leads to a warning being generated, this
+# script will remove the warning (if present) in the generated output.
+#
+# Can also be used with the environment variable (on bash)
+#
+#  export CHPL_SYSTEM_PREDIFF=$CHPL_HOME/util/test/prediff-for-incremental-warning
+
+import sys, subprocess
+import re
+
+# These are the tests that currently do not require the warning to be present in the output
+incrementalTests = ["time_iterate", "timeVectorArray", "time_access", "optimizeOnClauses_basic", "optimizeOnClauses_basic_record",
+                    "optimizeOnClauses_basic_tuple", "parSafeMember", "formalDomainChecks", "formalDomainChecks2", "access1d",
+                    "access2d", "access3d", "ri-4-arrdom", "ri-4-arrdom.replicated", "ri-3-stress", "ri-3-stress-coforall", "assign",
+                    "assign_across_locales", "copy_to_local", "dgemm", "init", "reductions-stress-fast", "sparse-iter-performance",
+                    "externMethodCallPerf", "forVersusWhilePerf", "test_cholesky_performance", "mandelbrot-error-works",
+                    "mandelbrot-error", "atomic_bool", "atomic_vars", "ontest"]
+
+if(sys.argv[1] not in incrementalTests):
+    sys.exit(0)
+
+# If the good file has a the warning for incremental compilation, the expected output should have it too
```

Comment:
Typo here: "If the good file has *a the* warning".  I'd fix this after the PR gets merged